manifests/crds/argoproj.io_applicationsets.yaml

```diff
@@ -1,13 +1,11 @@
+
 ---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.3.0
   creationTimestamp: null
-  labels:
```

Comment:
why delete this

Reply:
This is the autogenerated output from make generate, typically projects would let these be generated rather than maintained manually.

However was this file being maintained manually? It might explain the filename that confused me.

Or were these labels injected and I'm using an incorrect version of something.